File: www/MoneyGuardTypes/RiskStatus.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.RiskStatus = void 0;
var RiskStatus;
(function (RiskStatus) {
    RiskStatus[RiskStatus["RISK_STATUS_UNKNOWN"] = 0] = "RISK_STATUS_UNKNOWN";
    RiskStatus[RiskStatus["RISK_STATUS_SAFE"] = 1] = "RISK_STATUS_SAFE";
    RiskStatus[RiskStatus["RISK_STATUS_WARN"] = 2] = "RISK_STATUS_WARN";
    RiskStatus[RiskStatus["RISK_STATUS_UNSAFE"] = 3] = "RISK_STATUS_UNSAFE";
})(RiskStatus || (exports.RiskStatus = RiskStatus = {}));

File: src/android/model/RiskStatus.java
package com.wimika.ionic;

import com.google.gson.annotations.SerializedName;

public enum RiskStatus {
    @SerializedName("RISK_STATUS_UNKNOWN")
    RISK_STATUS_UNKNOWN,

    @SerializedName("RISK_STATUS_SAFE")
    RISK_STATUS_SAFE,

    @SerializedName("RISK_STATUS_WARN")
    RISK_STATUS_WARN,

    @SerializedName("RISK_STATUS_UNSAFE")
    RISK_STATUS_UNSAFE
}
